examples/cross-platform-with-console/index.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
```

Comment:
It doesn't make sense to start console and server at the same time, so I'd write something like this:

```js
const {
  ConsoleBot,
  MessengerBot,
  LineBot,
  TelegramBot,
  SlackBot,
  ViberBot,
} = require('bottender');

const handler = require('./handler');
const config = require('./bottender.config');

if (process.env.USE_CONSOLE === 'true') {
  const bot = new ConsoleBot().onEvent(handler);
  bot.createRuntime();
} else {
  const express = require('express');
  const bodyParser = require('body-parser');

  const { registerRoutes } = require('bottender/express');

  const server = express();

  server.use(
    bodyParser.json({
      verify: (req, res, buf) => {
        req.rawBody = buf.toString();
      },
    })
  );

  const bots = {
    messenger: new MessengerBot(config.messenger).onEvent(handler),
    line: new LineBot(config.line).onEvent(handler),
    slack: new SlackBot(config.slack).onEvent(handler),
    telegram: new TelegramBot(config.telegram).onEvent(handler),
    viber: new ViberBot(config.viber).onEvent(handler),
  };

  registerRoutes(server, bots.messenger, {
    path: '/messenger',
    verifyToken: config.messenger.verifyToken,
  });
  registerRoutes(server, bots.line, { path: '/line' });
  registerRoutes(server, bots.slack, { path: '/slack' });
  registerRoutes(server, bots.telegram, { path: '/telegram' });
  registerRoutes(server, bots.viber, { path: '/viber' });

  server.listen(5000, () => {
    console.log('server is listening on 5000 port...');
  });
}
```

Reply:
@chentsulin  Looks better.
But I prefer `npm run console` instead of using env var. What do you think?